MoreLinq.Test/TestingSequence.cs

```diff
@@ -69,14 +69,11 @@ public IEnumerator<T> GetEnumerator()
             _disposed = false;
             enumerator.Disposed += delegate
             {
-                Assert.That(_disposed, Is.False, "LINQ operators should not dispose a sequence more than once.");
```

Comment:
Per the [specification](https://learn.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose), it is not a failure to call `.Dispose()` multiple times. The `.Dispose()` method is expected to be idempotent, such that it is callable multiple times without throwing an exception.